eva-peng/week1/wed-homework-wordguesser/js/main.js

```diff
@@ -0,0 +1,66 @@
+// Homework: The Word Guesser
+
+// Create two global arrays: one to hold the letters of the word (e.g. 'F', 'O', 'X'), 
+// and one to hold the current guessed letters (e.g. it would start with '_', '_', '_' and end with 'F', 'O', 'X').
+
+const secretWordLetters = ['F', 'O', 'X'];
+const guessedLetters = ['_', '_', '_'];
+
+// Write a function called guessLetter that will: Take one argument, the guessed letter.
+
+
+const guessLetter = function (userGuess) {
+
+    let correctGuessMade = false;
+
+    for (let i = 0; i < secretWordLetters.length - 1; i++) {
```

Comment:
no need for the -1 behind secretWordLetters (thats why you are missing the X and its not working for x)

you want the for loop to be: `for (let i = 0; i < secretWordLetters.length; i++){}`
this means: `for (let i = 0; i < 3; i++){}`
so it goes through the loop like:

secretWordLetters[i] will be:
`secretWordLetters[0] = "F"`
`secretWordLetters[1] = "O"`
`secretWordLetters[2] = "X"`


but good work, you were so close!